catalog/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+from . import views
+from catalog import views
+
+urlpatterns = [
+    # path('', views.index, name='index'),
+    path('home/', views.home, name='home'),
```

Comment:
Лучше оставить корневую страницу, как выше
```suggestion
    path('', views.home, name='home'),
```